cpp/ps0001/main (2).cpp

```diff
@@ -2,9 +2,13 @@
 using namespace std;
 int main()
 {
-    short Value1,Value2,value3;
-    cout <<"Please enter the three values you wish to add together";
-    cin>>Value1>>Value2>>value3;
-    cout <<Value1+Value2+value3;
-return 0;
-}
\ No newline at end of file
+    // prompting user for 3 numbers to add 2/2
+    short value1, value2, value3;
+    cout << "Please enter the three values you wish to add together: \n";
+    cin >> value1 >> value2 >> value3;
```

Comment:
- [x] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add